rust/hyperlane-core/src/chain.rs

```diff
@@ -258,16 +228,17 @@ impl KnownHyperlaneDomain {
 
         many_to_one!(match self {
             HyperlaneDomainProtocol::Ethereum: [
-                Ethereum, Sepolia, Holesky, Polygon, Avalanche, Fuji, Arbitrum,
-                Optimism, BinanceSmartChain, BinanceSmartChainTestnet, Celo, Gnosis,
-                Alfajores, Moonbeam, InEvm, Ancient8, Blast, Mode, Redstone, Viction,
-                Zetachain, MoonbaseAlpha, ScrollSepolia, Chiado, MantaPacific, PlumeTestnet,
-                Fraxtal, Linea, Sei, Bob, Mantle, Taiko,
-                Test1, Test2, Test3
+                Alfajores, Ancient8, Arbitrum, Avalanche, BinanceSmartChain,
+                BinanceSmartChainTestnet, Blast, Bob, Celo, Chiado, Ethereum, Fraxtal, Fuji,
+                Gnosis, Holesky, InEvm, Linea, MantaPacific, Mantle, Mode, MoonbaseAlpha, Moonbeam,
+                Optimism, PlumeTestnet, Polygon, Redstone, ScrollSepolia, Sei, Sepolia, Taiko,
+                Test1, Test2, Test3, Viction, Zetachain,
             ],
             HyperlaneDomainProtocol::Fuel: [FuelTest1],
             HyperlaneDomainProtocol::Sealevel: [SealevelTest1, SealevelTest2],
-            HyperlaneDomainProtocol::Cosmos: [CosmosTest99990, CosmosTest99991, Neutron, Osmosis, Injective],
+            HyperlaneDomainProtocol::Cosmos: [
+                CosmosTest99990, CosmosTest99991, Injective, Neutron, Osmosis
```

Comment:
would be great to separate by mainnet/testnet here (just a comment dividing them is fine, similar to how the top-level enum is ordered)

Reply:
ditto lines 251-257